code/game/jobs/job/job.dm

```diff
@@ -4,6 +4,7 @@
 	var/title = "NOPE"
 	var/list/access = list()				// Useful for servers which either have fewer players, so each person needs to fill more than one role, or servers which like to give more access, so players can't hide forever in their super secure departments (I'm looking at you, chemistry!)
 	var/list/cruciform_access = list()		// Assign this access into cruciform if target has it
+	var/security_clearance = 0				// Cruciform-specific access type, used by neotheologian doors
```

Comment:
Instead of numbers here, I would use defines like `CLEARANCE_NONE = 0 CLEARANCE_COMMON = 1 CLEARANCE_CLERGY = 2` so the code is more readable.